src/components/home/Hero/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { Typography } from '@/components/common';
+import { config } from '@/lib';
+import type { PrivateProfile } from '@/lib/types/apiResponses';
+import CheckMark from '@/public/assets/icons/check-mark.svg';
+import LeaderboardIcon from '@/public/assets/icons/leaderboard-icon.svg';
+import ShopIcon from '@/public/assets/icons/shop-icon.svg';
+import Image from 'next/image';
+import Link from 'next/link';
+import { FormEvent, useState } from 'react';
+import styles from './style.module.scss';
+
+interface HeroProps {
+  user: PrivateProfile;
+  checkin: (code: string) => void;
+}
+
+const Hero = ({ user, checkin }: HeroProps) => {
+  const [checkinCode, setCheckinCode] = useState('');
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    checkin(checkinCode);
+  };
+
+  return (
+    <div className={styles.hero}>
+      <Image
+        className={`${styles.image} ${styles.desktop}`}
+        src="/assets/graphics/portal/raccoon-waves.svg"
+        alt="Landing page graphic"
+        priority
+        fill
+      />
+      <Image
+        className={`${styles.image} ${styles.mobile}`}
+        src="/assets/graphics/portal/waves.svg"
+        alt="Landing page graphic"
+        priority
+        fill
+      />
```

Comment:
you could also remove the waves from the raccoon image so you can keep waves.svg visible always, and it allows for the raccoon to move more independently if we want to